src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should be more informative. Instead of 'Bulldozer stopped working', consider 'Bulldozer has stopped working' to maintain consistency and clarity.